packages/jazz-react/src/scratch.tsx

```diff
@@ -0,0 +1,19 @@
+import { useAccount, useIsAuthenticated } from "jazz-react";
```

Comment:
Is this some leftover?

Reply:
Whoops, that's just a file for me to test stuff in, I'll remove it